src/content/docs/en/guides/view-transitions.mdx

```diff
@@ -356,7 +356,27 @@ For example, if you are implementing dark mode support, this event can be used t
 </script>
 ```
 
-## `prefers-reduced-motion`
+## Accessibility
+
+Enabling client-side routing and animating page transitions both come with accessibility challenges, and Astro aims to make sites opting into View Transitions as accessible-by-default as possible.
```

Comment:
```suggestion
Enabling client-side routing and animating page transitions both come with accessibility challenges, and Astro aims to make sites opting in to View Transitions as accessible-by-default as possible.
```
We're going to get cards and letters, even from Yan, but this is 100% correct. :sweat_smile: